test/data/testPFalg.jl

```diff
@@ -0,0 +1,117 @@
+using Pkg
```

Comment:
some here, don't forget to remove this file later. if you want to keep it, morph it into a Pluto notebook